### ISSUE TITLE
distributed 2.9.0 has requirement dask>=2.7.0

### ISSUE THREAD

**Issue by raar1** [UCL-CCS/EasyVVUQ#175]
Just noticed the following when attempting to install from requirements.txt. This happens on both my laptop and on travis:

```
distributed 2.9.0 has requirement dask>=2.7.0, but you'll have dask 2.4.0 which is incompatible.
```

It doesn't cause any tests to fail, but it's concerning. Any idea what this is referring to? (@orbitfold maybe?) Is there any reason we can't set dask==2.7.0 at present?

**Comment by raar1** [comment]
In order to fix Issue #177 I needed to raise Dask's version requirement to 2.7.0. The tests all still pass, and it avoids the problem in this issue.

Is there some reason we have to have Dask at 2.4.0?

**Comment by orbitfold** [comment]
No, but wasn't there something about us wanting exact versions rather than >=? That's why I fixed the version to an arbitrary one that worked for me. But feel free to change if it works.

**Comment by raar1** [comment]
Yeah, I've fixed it to 2.7.0 now. Should fix this issue.
